modules/examples/todolist-service/src/main/scala/todo/runtime/queries/AppQueries.scala

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
I think all these queries could be placed inside of the `persistence` package => `todo.persistence.queries`

Does it make sense to you?

Reply:
Since queries have a dependency with `doobie`, I thought they should be in the same package as `handlers`.

Reply:
👍 Sounds good